backend/test/resolvers/portfolioPeriodQuery.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from "chai";
+
+import { openPortfolioPeriod } from "../../resolvers/queries/portfolioPeriodQuery";
+import { fakeUser, fakePortfolioPeriod, fakePortfolio } from "../factories";
+
+describe("Portfolio Period Queries", function() {
+  describe("openPortfolioPeriod", function() {
```

Comment:
This tests the new open portfolio period query which pulls the open period only when the student does not yet have a portfolio for it.